tests/quantization/torchao/test_torchao.py

```diff
@@ -131,8 +131,9 @@ def tearDown(self):
         gc.collect()
         torch.cuda.empty_cache()
 
-    def get_dummy_components(self, quantization_config: TorchAoConfig):
-        model_id = "hf-internal-testing/tiny-flux-pipe"
+    def get_dummy_components(
+        self, quantization_config: TorchAoConfig, model_id: str = "hf-internal-testing/tiny-flux-pipe"
```

Comment:
Changed here because I want both sharded and non-sharded to be tested in some of the tests below. The tests will call this method with two different model ids

Reply:
Nice that we're leveraging a tiny model here for the sharded state dict as well:
hf-internal-testing/tiny-flux-sharded